pyproject.toml

```
@@ -0,0 +1,3 @@
+[build-system]
+requires = ["jupyterlab>=3.0.0", "setuptools>=40.8.0", "wheel"]
```

We should probably use, since this is unlikely to work with lab 4 ?:
```suggestion
requires = ["jupyterlab~=3.0", "setuptools>=40.8.0", "wheel"]
```

---

That's a good suggestion, I'll constrain that.